recipes-graphics/libepoxy/libepoxy_%.bbappend

```diff
@@ -2,10 +2,10 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 SRC_URI += "file://0001-Make-graphic-libs-configurable.patch"
 
-GLX_LIB_NAME    ?= "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/libgl',    'broadcom-refsw', 'libv3ddriver.so', 'libGL.so.1', d)}"
-EGL_LIB_NAME    ?= "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/egl',      'broadcom-refsw', 'libv3ddriver.so', 'libEGL.so.1', d)}"
-GLES1_LIB_NAME  ?= "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/libgles1', 'broadcom-refsw', 'libv3ddriver.so', 'libGLESv1_CM.so.1', d)}"
-GLES2_LIB_NAME  ?= "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/libgles2', 'broadcom-refsw', 'libv3ddriver.so', 'libGLESv2.so.2', d)}"
+GLX_LIB_NAME    ?= "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/libgl',    'broadcom-refsw', 'libv3ddriver.so', '/usr/lib/libGL.so.1', d)}"
+EGL_LIB_NAME    ?= "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/egl',      'broadcom-refsw', 'libv3ddriver.so', '/usr/lib/libEGL.so.1', d)}"
+GLES1_LIB_NAME  ?= "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/libgles1', 'broadcom-refsw', 'libv3ddriver.so', '/usr/lib/libGLESv1_CM.so.1', d)}"
+GLES2_LIB_NAME  ?= "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/libgles2', 'broadcom-refsw', 'libv3ddriver.so', '/usr/lib/libGLESv2.so.2', d)}"
```

Comment:
This addresses the same issue as #301 on rpi3.

Reply:
yes it does, although i think the patch in meta-wpe is misplaced in general but thats a different issue

Reply:
@psivasubramanian  @kraj Whether it includes the full path or not is irrelevant I think. I believe is the "double quotes" (see strace output in https://github.com/WebPlatformForEmbedded/meta-wpe/issues/301) that cause the open syscall to fail. I tried with and without the full path and it always fails even though the library is present at that location.

Reply:
@petegriffin Using full path had helped in Rpi platform here, that why we propose this change.